wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights Followers card

Fixes #10668 

---
Notes: 
- The total followers is not abbreviated due to [this issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/10562) (Follower totals are currently only available in string format).
- `View more` shows a temporary alert.

---
To test:

- On a site with WP/Email followers, go to Stats > Insights.
  - Tip: Hogwarts and en.blog are good examples.
- Scroll to `Followers`.
- Verify:
  - The total count is shown on tabs with data.
  - `No data yet` is displayed on tabs with no data.
  - `View more` is shown if there are more than 6 followers.
  - If neither tab has data, the `FilterTabBar` is hidden.
    - Hack tip: to emulate this, in `SiteStatsInsightsViewModel:tabDataForFollowerType`, when it calls `TabData.init`, set `dataRows` to an empty array.

----
<img width="350" alt="followers" src="https://user-images.githubusercontent.com/1816888/50189992-c278b380-02e4-11e9-83a0-95c59ba3dd3a.png">

---
<img width="350" alt="no_followers" src="https://user-images.githubusercontent.com/1816888/50190010-d2909300-02e4-11e9-8b72-b9de7c37e1e2.png">

---
<img width="350" alt="no_tab_data" src="https://user-images.githubusercontent.com/1816888/50248619-d8e04700-0398-11e9-9c17-da2a75dfe301.png">
